sdks/python/apache_beam/runners/direct/watermark_manager.py

```diff
@@ -94,14 +95,14 @@ def get_watermarks(self, applied_ptransform):
 
   def update_watermarks(self, completed_committed_bundle, applied_ptransform,
                         completed_timers, outputs, unprocessed_bundles,
-                        keyed_earliest_holds):
+                        keyed_earliest_holds, side_inputs_container):
```

Comment:
How is this new argument `side_inputs_container` different from the one passed to the constructor above?

Can you use `self._side_inputs_container` instead of adding this new argument here?

Reply:
You're right--Maria, can you fix this?

Reply:
I already pushed the change earlier, sorry I forgot to comment here.
Actually what I don't need is the  attribute, since I got rid of it.